src/cljs/proton/layers/base.cljs

```diff
@@ -10,3 +12,43 @@
 (defmulti get-keybindings dispatch)
 (defmulti get-keymaps dispatch)
 (defmulti describe-mode dispatch)
+
+(defn register-package-dependencies [key deps]
```

Comment:
I think it makes sense to tie layer dependencies to a complete layer, not just a package. I'm in favor of removing